templates/event/sponsor-request.html

```diff
@@ -0,0 +1,33 @@
+{% extends 'global/base.html' %}
+{% load staticfiles %}
+
+{% block title %}Request for GitHub sponsoring{% endblock %}
+{% block description %}Fill the form available on this page to apply for a GitHub grant.{% endblock %}
+
+{% block content %}
+    <section id="flatpage">
+        <div class="row clearfix">
+            <div class="col-md-8 col-md-offset-2 col-sm-10 col-sm-offset-1 col-xs-10 col-xs-offset-1">
+                <h1>Request for GitHub sponsoring</h1>
+
+                <p>GitHub can sponsor one Django Girls workshop per city with a 500$ grant.</p>
+
+                <p>Before applying, your website needs to be ready and online. Then, you need to
+                    <a href="https://djangogirls.org/events/">check if an event already happened in your city</a>
```

Comment:
Can you use the `{% url ... %}` tag here instead of `https://djangogirls.org/events/`. Look inside the `urls.py` file for the name of the URL to use.